src/generic/linear_solver.h

```diff
@@ -645,6 +645,13 @@ namespace oomph
     /// See enum SuperLU_solver_type for more details.
     void set_solver_type(const Type& t)
     {
+      // hierher
+      if (t == Distributed)
+      {
+        oomph_info << "SuperLU_dist is currently disabled" << std::endl;
```

Comment:
Same logic: As you said, SuperlU dist may better on cuda etc. so it would be nice to keep it as an option. In this sense it's disabled -- kept alive until we decide to get rid of it for good at which point it becomes deprecated and kill the lot.

Reply:
Ah! I stand corrected then